plugin.video.itvrs/resources/lib/itvrs.py

```diff
@@ -0,0 +1,86 @@
+
+# -*- coding: utf-8 -*-
+#------------------------------------------------------------
+# License: GPL (http://www.gnu.org/licenses/gpl-3.0.html)
+# Based on code from youtube addon
+#------------------------------------------------------------
+
+import os
+import sys
+import plugintools
+import xbmc,xbmcaddon
+from addon.common.addon import Addon
+
+addonID = 'plugin.video.itvrs'
+addon = Addon(addonID, sys.argv)
+local = xbmcaddon.Addon()
+icon = local.getAddonInfo('icon')
+
+YOUTUBE_CHANNEL_ID_1 = "UCB2U9Zvptf24QzBnNKHUtsw"
+
+# Entry point
+def run():
+    plugintools.log("itvrs.run")
+    
+    # Get params
+    params = plugintools.get_params()
+    
+    if params.get("action") is None:
+        main_list(params)
+    else:
+        pass
+    
+    plugintools.close_item_list()
+
+# Main menu
+def main_list(params):
+    plugintools.log("itvrs.main_list "+repr(params))
+    
+    plugintools.add_item( 
+        #action="", 
+        title="ITVRS Live TV",
+        url="https://www.youtube.com/embed/live_stream?channel="+YOUTUBE_CHANNEL_ID_1+"/",
+        thumbnail=icon,
```

Comment:
It would be nice if you could check what the `plugintools.add_item`is doing in the background. `thumbnailImage` and `iconImage` usage in the `xbmcgui.ListItem` constructor will result in NOP for Matrix and above.